WcaOnRails/app/assets/stylesheets/navbar-static-top.scss

```diff
@@ -56,23 +36,29 @@
     margin-bottom: -($nav-avatar-thumbnail-size - 20px) / 2;
   }
 
+  .search-container{
+    width: 100%;
+    padding: 10px;
+  }
+
+
   .navbar-collapse {
-    background: #ddd;
     margin: 0;
     clear: both;
 
     $omni-search-sm-width: 230px;
-    $omni-search-md-width: 450px;
+    $omni-search-md-width: 300px;
     $omni-search-lg-width: 500px;
 
+
     form.navbar-form[role="search"] .wca-autocomplete-omni + .selectize-control .selectize-input {
       @media (min-width: $screen-sm-min) {
         width: $omni-search-sm-width;
       }
-      @media (min-width: $screen-md-min) {
+      @media (min-width: 1000) {
```

Comment:
It should be `px` here (I think otherwise it's treated as invalid and ignored).

Also, do we really need those fixed values rather than using the predefined breakpoints? (it's ok if we do)

Reply:
Yes, I added my own breakpoints so the search bar would not overlap the search bar. I ended up changing the width to 1200px - thanks for the suggestion!